config.go

```diff
@@ -625,6 +631,7 @@ func DefaultConfig() Config {
 		PendingCommitInterval:   defaultPendingCommitInterval,
 		ChannelCommitBatchSize:  defaultChannelCommitBatchSize,
 		CoinSelectionStrategy:   defaultCoinSelectionStrategy,
+		KeepFailedPaymentAttempts:      defaultKeepFailedPaymentAttempts,
```

Comment:
Looks like this file should be formatted... Not sure why the linter didn't pick this up.

Reply:
Done. Not sure how that slipped by...